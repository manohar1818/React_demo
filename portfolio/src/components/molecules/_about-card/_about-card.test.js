import React from 'react';
import { create } from "react-test-renderer";
import { render ,fireEvent} from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import AboutCard from './_about-card';
 
const array=["Spring","React"]
describe("AboutCard component",() =>{
    test("Matches the snapshot",() => {
        const  aboutCard= create(<AboutCard heading="Technical Skills" array={array}/>);
        expect(aboutCard.toJSON()).toMatchSnapshot();

    })
    test("about card element is rendered",() => {
        const { getByTestId} = render(<AboutCard heading="Technical Skills" array={array}/>);
        expect(getByTestId("aboutCard")).toBeInTheDocument();
        
    });

  
});