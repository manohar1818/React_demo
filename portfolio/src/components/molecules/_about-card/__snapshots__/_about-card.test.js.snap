// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutCard component Matches the snapshot 1`] = `
<div
  className="makeStyles-list-1"
  data-testid="aboutCard"
>
  <h6
    className="MuiTypography-root makeStyles-heading-2 MuiTypography-subtitle1"
  >
    Technical Skills
  </h6>
  <p
    className="MuiTypography-root MuiTypography-body1"
  >
    Spring
  </p>
  <p
    className="MuiTypography-root MuiTypography-body1"
  >
    React
  </p>
</div>
`;
